projects/Zenlink/links_v1.json

```diff
@@ -0,0 +1,9 @@
+{
+	"website": "https://zenlink.pro/",
+	"twitter": "https://twitter.com/zenlinkpro",
+	"github": "https://github.com/zenlinkpro",
+	"develop_doc": "",
+	"element": "",
+	"telegram": "https://t.me/ZenlinkPro_EN",
+	"discord": ""
```

Comment:
If you don't have some of items, you could cancel directly

Reply:
Done